CRM/Member/Form/Membership.php

```diff
@@ -1685,7 +1679,8 @@ public function submit() {
         }
         if ($taxAmount) {
           $this->assign('totalTaxAmount', $totalTaxAmount);
-          $this->assign('taxTerm', CRM_Utils_Array::value('tax_term', $invoiceSettings));
+          // Not sure why would need this on Submit.... unless it's being used when sending mails in which case this is the wrong place
```

Comment:
@mattwire added as a code comment

Reply:
I'm happy :-)

Reply:
Lol - let me know if you feel this is mergeable